### ISSUE TITLE
Feature request: Show active Sprinkler Name in "Info" object

### ISSUE THREAD

**Issue by cyberax17** [Dirk-Peter-md/ioBroker.sprinklecontrol#121]
Hi, for me ist would het Great of there was an status object  in the Info folder which tells you the running sprinkler(s) names. Or is a similar feature implemented already ? Cheers and many thanks for this great adapter!

**Comment by Dirk-Peter-md** [comment]
Hallo, ich nehme an du brauchst dies für die Verarbeitung in der  VIS! Wie sollte es da aussehen für die bessere Verarbeitung. JSON? Habe leider mit der VIS noch keine Erfahrung. 
Beispiel:
`
[
{"name":"Auffahrt", "Bodenfeuchte":"62.8% (70%)", "Dauer": "8:00", "Status":"on"},
{"name":"Vorgarten", "Bodenfeuchte":"75.9% (50%)", "Dauer": "00:00", "Status":"off"},
{"name":"Blumenbeet", "Bodenfeuchte":"75.9% (50%)", "Dauer": "32:13", "Status":""},
{"name":"Rasenumrandung", "Bodenfeuchte":"46.3% (50%)", "Dauer": "00:00", "Status":"off"},
{"name":"Koniferen", "Bodenfeuchte":"75.9% (50%)", "Dauer": "00:00", "Status":"off"},
{"name":"RasenflÃ¤che_am_Beet", "Bodenfeuchte":"48.6% (50%)", "Dauer":"20:33", "Status":"wait"}
{"name":"RasenflÃ¤che_am_Haus", "Bodenfeuchte":"48.8% (50%)", "Dauer":"20:30", "Status":"wait"}, 
{"name":"Schattenrasen_am_Haus", "Bodenfeuchte":"32.2% (50%)", "Dauer":"27:08", "Status":"on"}, 
{"name":"Schattenrasen_an_den_Koniferen", "Bodenfeuchte":"32.2% (50%)", "Dauer":"27:08", "Status":"wait"}, 
{"name":"Steingarten_am_Carport", "Bodenfeuchte":"75.2% (50%)", "Dauer": "00:00", "Status":"off"} 
]
`

Kann nichts versprechen, aber wenn so kommt es erst zur nächsten Gartensaison, Dirk Peter

**Comment by cyberax17** [comment]
Yes the idea is for visualisation purposes. My dashboard is in NodeRed. JSON would be fine in deed. In my case it could also be more than one circuits running at the same time depending on the pump/consumption budget.
Next irrigation season would be sufficient 👍🙏

Hätte auch auf Deutsch schreiben können, dachte aber so ist es internationaler 😃
